examples/offloading-with-policies/manifests/testing-deploy.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
```

Comment:
Would rename this file to `deploy.yaml`, as testing-deploy is longer, ugly (at least IMHO) and adds no value.